src/cow-react/modules/limitOrders/hooks/useLimitOrdersFullState.ts

```diff
@@ -0,0 +1,24 @@
+import { useAtomValue, useUpdateAtom } from 'jotai/utils'
+import {
+  limitOrdersAtom,
+  LimitOrdersFullState,
+  limitOrdersFullStateAtom,
+} from '@cow/modules/limitOrders/state/limitOrdersAtom'
+import { useEffect } from 'react'
+import { useBuildTradeFullState } from '@cow/modules/trade/hooks/useBuildTradeFullState'
+
+export function useLimitOrdersFullState(): LimitOrdersFullState {
+  return useAtomValue(limitOrdersFullStateAtom)
+}
+
+export function useFillLimitOrdersFullState() {
+  const rawState = useAtomValue(limitOrdersAtom)
```

Comment:
Can we just return `isUnlocked` from `useBuildTradeFullState` so that we don't have to call this here?

Reply:
Unfortunately no, `useBuildTradeFullState` is a generic function that can be used for any trade state.
Anyway, I guess we should remove `isUnlocked` soon